rolling-shutter/keyperimpl/gnosis/config.go

```diff
@@ -0,0 +1,72 @@
+package gnosis
+
+import (
+	"io"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/keyper/kprconfig"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/configuration"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/metricsserver"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/p2p"
+)
+
+var _ configuration.Config = &Config{}
+
+func NewConfig() *Config {
+	c := &Config{}
+	c.Init()
+	return c
+}
+
+func (c *Config) Init() {
+	c.P2P = p2p.NewConfig()
+	c.Gnosis = configuration.NewEthnodeConfig()
+	c.Shuttermint = kprconfig.NewShuttermintConfig()
+	c.Metrics = metricsserver.NewConfig()
+}
+
+type Config struct {
+	InstanceID  uint64 `shconfig:",required"`
+	DatabaseURL string `shconfig:",required" comment:"If it's empty, we use the standard PG_ environment variables"`
+
+	HTTPEnabled       bool
+	HTTPListenAddress string
+
+	P2P         *p2p.Config
+	Gnosis      *configuration.EthnodeConfig
+	Shuttermint *kprconfig.ShuttermintConfig
+	Metrics     *metricsserver.MetricsConfig
+}
+
+func (c *Config) Validate() error {
+	return nil
+}
+
+func (c *Config) Name() string {
+	return "keyper"
```

Comment:
Here you can change the name to represent the CLI command name you want.
So e.g. `return "gnosiskeyper"` in your case. 
This is a little bit different than chosing the `CommandName` option above and sticks to the convention of other keyperimpl's